bin/ruby-build

```diff
@@ -154,6 +154,10 @@ num_cpu_cores() {
   echo "${num:-2}"
 }
 
+install_package_direct() {
```

Comment:
@mislav understood, i just reverted to the original method of ad-hoc check with the cache.ruby-lang.org. the improvements to test and doc are still in place.